### PR TITLE
Issue 7281: batch delete snapshot

### PR DESCRIPTION
**Pull request by Lyndon-Li**
Fix issue #7281, batch delete snapshots in the same repo

### CODE REVIEW

**Review by ywk253100 — DISMISSED** [review]

File: pkg/datamover/dataupload_delete_action.go [hunk 1]
```diff
@@ -57,6 +57,8 @@ func genConfigmap(bak *velerov1.Backup, du velerov2alpha1.DataUpload) *corev1api
 		BackupStorageLocation: bak.Spec.StorageLocation,
 		SnapshotID:            du.Status.SnapshotID,
 		RepositoryType:        GetUploaderType(du.Spec.DataMover),
```
Comment (blackpiglet) [inline pkg/datamover/dataupload_delete_action.go:59]: Although this line is unrelated to this PR change, this line does not look right. 
It should be something like "kopia".

Comment (Lyndon-Li) [reply pkg/datamover/dataupload_delete_action.go]: Yes, I have modified it, thanks.

**Review by blackpiglet — APPROVED** [review]

**Review by ywk253100 — APPROVED** [review]
